src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -14,7 +14,7 @@ public sealed partial class Utf8JsonWriter
         /// Writes the pre-encoded property name and raw bytes value (as a base 64 encoded JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
         /// <param name="propertyName">The JSON-encoded name of the property to write.</param>
-        /// <param name="bytes">The Base64-encoded data to write.</param>
+        /// <param name="bytes">The binary data to write as a Base64-encoded text.</param>
```

Comment:
Looks like we use "base-64" or base 64 in existing docs (lower case and with either a hyphen or space in the middle).
https://docs.microsoft.com/en-us/dotnet/api/system.convert.frombase64string?view=netcore-3.0
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.text.base64?view=netcore-3.0

Let's be consistent here too.

```suggestion
        /// <param name="bytes">The binary data to write as a base 64 encoded text.</param>
```

The summary/etc. also have it as `base 64`

Reply:
Actually I think I just discussed this with @rpetrusha and we thought that Base64 was the correct one. So if we should fix other instances in the docs that are not using this.

Reply:
Here's where we did taht: https://github.com/dotnet/dotnet-api-docs/pull/2890/commits/0effaf77a07bfe430858c4cc516add391941bb97

Reply:
What about the inconsistency with the existing API docs then, that I linked above?

Reply:
I put up a PR to be consistent with the fixes from this PR as well: https://github.com/dotnet/dotnet-api-docs/pull/2976

Reply:
I’ll fix the other places that are inconsistent as well